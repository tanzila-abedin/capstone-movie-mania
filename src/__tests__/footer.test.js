// import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Footer from '../Components/footer';

test('footer component is defined', async () => {
  const component = render(
    <Footer />,
  );
  const footer = await component.findAllByTestId('footer');
  expect(footer).toBeDefined();
});
