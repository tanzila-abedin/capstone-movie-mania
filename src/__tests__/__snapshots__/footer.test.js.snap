// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer snapshot with react and redux 1`] = `
<nav
  className="navbar navbar-expand-sm navHead navHead  fixed-bottom"
  data-testid="footer"
>
  <div
    className="container text-center text-light d-flex justify-content-center"
  >
    Built By:
    <span
      className="text-warning font-weight-normal"
    >
      Tanzila Abedin
    </span>
    , Using
     
    <a
      className="text-warning "
      href="https://imdb-api.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      IMDb
    </a>
    API
  </div>
</nav>
`;
